### PR TITLE
Screenshot delay

### LINKED ISSUE #80: Increase delay before making a screenshot
* Lost pixel screenshot delay
* Playwright screenshot delay

### PR DESCRIPTION
**Pull request by arthur-kashaiev-ssa-group**
Closes #80

### CODE REVIEW

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 4  | ❗       |
    


*❗ storybook/components-popover--default__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/components-popover--default__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/components-popover--default__[w390px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/components-popover--default__[w900px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 4       | 🟡       |
| Deleted         | 4     | ❌       |
| Regression      | 8  | ❗       |
    


*🟡 custom/widgets-navbar-opened__[w1440px]*: 


| Image |
| --- |
|  |



*🟡 custom/widgets-navbar-opened__[w1920px]*: 


| Image |
| --- |
|  |



*🟡 custom/widgets-navbar-opened__[w390px]*: 


| Image |
| --- |
|  |



*🟡 custom/widgets-navbar-opened__[w900px]*: 


| Image |
| --- |
|  |



*❌ custom/widgets-navbar--mobile-opened__[w1440px]*: 


| Image |
| --- |
|  |



*❌ custom/widgets-navbar--mobile-opened__[w1920px]*: 


| Image |
| --- |
|  |



*❌ custom/widgets-navbar--mobile-opened__[w390px]*: 


| Image |
| --- |
|  |



*❌ custom/widgets-navbar--mobile-opened__[w900px]*: 


| Image |
| --- |
|  |



<details>
<summary>❗ custom/widgets-heart-rate--hint-opened__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-heart-rate--hint-opened__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-heart-rate--hint-opened__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-heart-rate--hint-opened__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-heart-rate--default__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-heart-rate--default__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-meal-nutrients--default__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-progress-info--default__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 2       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 4  | ❗       |
    


*🟡 custom/widgets-navbar-opened__[w1920px]*: 


| Image |
| --- |
|  |



*🟡 custom/widgets-navbar-opened__[w900px]*: 


| Image |
| --- |
|  |



*❗ custom/widgets-heart-rate--hint-opened__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-navbar-opened__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-navbar-opened__[w390px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/widgets-heart-rate--default__[w900px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 2  | ❗       |
    


*❗ storybook/utility-responsive-image--responsive-image__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/utility-responsive-image--responsive-image__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 1       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 4  | ❗       |
    


*🟡 custom/widgets-navbar-opened__[w900px]*: 


| Image |
| --- |
|  |



*❗ custom/widgets-heart-rate--hint-opened__[w900px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-navbar-opened__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-navbar-opened__[w390px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/widgets-progress-info--default__[w390px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 0  | ❗       |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 1       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 4  | ❗       |
    


*🟡 custom/widgets-navbar-opened__[w900px]*: 


| Image |
| --- |
|  |



*❗ custom/widgets-navbar-opened__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-navbar-opened__[w390px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/widgets-heart-rate--default__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/widgets-progress-info--default__[w390px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 16  | ❗       |
    


<details>
<summary>❗ storybook/components-card--item-card__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-action__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-action__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-action__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-action__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-avatar__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-avatar__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-avatar__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-avatar__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/utility-responsive-image--responsive-image__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/utility-responsive-image--responsive-image__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/utility-responsive-image--responsive-image__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/utility-responsive-image--responsive-image__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 13       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 11  | ❗       |
    


<details>
<summary>🟡 custom/widgets-meal-nutrients--hint-opened__dropdown__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-meal-nutrients--hint-opened__dropdown__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-meal-nutrients--hint-opened__dropdown__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-meal-nutrients--hint-opened__dropdown__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-navbar-opened__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-progressInfo__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-progressInfo__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-progressInfo__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-progressInfo__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-progressInfo__dropdown__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-progressInfo__dropdown__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-progressInfo__dropdown__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-progressInfo__dropdown__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>❗ custom/widgets-heart-rate--hint-opened__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-heart-rate--hint-opened__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-heart-rate--hint-opened__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-heart-rate--hint-opened__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-meal-nutrients--hint-opened__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-navbar-opened__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-heart-rate--default__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-progress-info--default__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-progress-info--default__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-progress-info--default__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-progress-info--default__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 2  | ❗       |
    


*❗ storybook/utility-responsive-image--responsive-image__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/utility-responsive-image--responsive-image__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 12       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 8  | ❗       |
    


<details>
<summary>🟡 storybook/components-indicator--default__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-indicator--default__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-indicator--default__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-indicator--default__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-indicator--right-side__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-indicator--right-side__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-indicator--right-side__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-indicator--right-side__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-indicator--with-content__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-indicator--with-content__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-indicator--with-content__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-indicator--with-content__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>❗ storybook/components-button--all-states__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-button--all-states__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-button--all-states__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-button--all-states__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/utility-responsive-image--responsive-image__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/utility-responsive-image--responsive-image__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/utility-responsive-image--responsive-image__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/utility-responsive-image--responsive-image__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 8  | ❗       |
    


<details>
<summary>❗ custom/widgets-meal-nutrients--hint-opened__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-navbar-opened__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-navbar-opened__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-heart-rate--default__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-progress-info--default__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-progress-info--default__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-progress-info--default__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-progress-info--default__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 2  | ❗       |
    


*❗ storybook/components-indicator--with-content__[w900px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/utility-responsive-image--responsive-image__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 6  | ❗       |
    


<details>
<summary>❗ custom/widgets-meal-nutrients--hint-opened__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-meal-nutrients--hint-opened__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-navbar-opened__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-heart-rate--default__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-heart-rate--default__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-steps-counter--default__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by arthur-kashaiev-ssa-group** [comment]
This is not a 100% solution because we need to use only screenshots from (e2e tests) Playwright to get the solution we want.
But it's enough to close #71 and #80 for now.

File: lostpixel.base.config.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { CustomProjectConfig } from 'lost-pixel';
```
Comment (react-developer-ssa-group) [inline lostpixel.base.config.ts:1]: We have a lot of configs in the project root. Probably, we need additional task to move some of them to the common directory ("config" for example)

**Review by react-developer-ssa-group — APPROVED** [review]

**Pull request merged**